crunchbase/provider/client.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from flask import current_app as app
+
+from . import UpstreamProviderError
+
+client = None
+
+
+class CrunchbaseApiClient:
+    API_URL = "https://api.crunchbase.com/api/v4/autocompletes"
+
+    def __init__(self, api_key, search_limit):
+        self.user_key = api_key
+        self.search_limit = search_limit
+
+    def get_search_limit(self):
```

Comment:
Added a getter here but it is not being used. I would just remove this method, since you're calling ` self.search_limit` later on which is pythonic.

Reply:
In general - fields in python are accessible by default and I haven't seen a good use case to add a getter and setter _unless_ some preprocessing needs to happen, accessing the attribute directly is completely fine